docs/intro/tutorial.rst

```diff
@@ -277,9 +277,9 @@ As an alternative, you could've written:
 >>> response.css('title::text')[0].get()
 'Quotes to Scrape'
 
-However, using ``.get()`` directly on a :class:`~scrapy.selector.SelectorList`
-instance avoids an ``IndexError`` and returns ``None`` when it doesn't
-find any element matching the selection.
+Directly accessing an index on a :class:`~scrapy.selector.SelectorList`
+instance could potentially run into an ``IndexError``. It is recommended to use
+``.get()`` directly instead as it avoids such index errors.
```

Comment:
I see your point. What about this?

```suggestion
Accessing an index on a :class:`~scrapy.selector.SelectorList` instance will 
raise an :exc:`IndexError` exception if there are no results::

    >>> response.css('noelement')[0].get()
    Traceback (most recent call last):
    File "<console>", line 1, in <module>
    ...
    IndexError: list index out of range

You might want to use ``.get()`` directly on the 
:class:`~scrapy.selector.SelectorList` instead, which returns ``None`` if there 
are no results::

>>> response.css("noelement").get()
```

Reply:
Looks perfect to me! Definitely clears up any issue I had with it.